### ISSUE TITLE
Casing to uppercase does not work when adding input or output parameter to existing method or function

### ISSUE THREAD

**Issue by Ton-Pe** [omecra/de.omecra.oedt.updatesite#194]
After updating from OEDT 9.3.5 we noticed following bug.
Casing is set to uppercase.
![image](https://github.com/user-attachments/assets/ceb3fa8e-4b9e-4fa6-bcfd-f7dccf5fa48c)

Add new input, or output, parameter to existing method or function. Neither input keyword nor output keyword gets changed to uppercase. Those still stay as lowercase. (datatype and as keywords are changed to uppercase).

``` 
METHOD PUBLIC VOID MyMethod (INPUT i AS INTEGER, input iFoo AS INTEGER, output oFoo AS LOGICAL ):
   
END METHOD.
```
Then try to start writing a new method and input/output keywords are changed automatically to uppercase.
```
METHOD MyNewMethod (INPUT iFoo AS INTEGER, OUTPUT oFoo AS LOGICAL)
```

I noticed that right after I enter the colon at the end of method definition, the automatic casing stops working!

Correct keywords action works and changes those to uppercase. I have tried setting "Use smart Keyword casing when typing" on and off, but no effect.

OEDT version 9.5.5
PDSOE version 12.8.3

**Comment by lneumeier** [comment]
Will be fixed in OEDT 9.5.6.

**Comment by lievendf** [comment]
> Will be fixed in OEDT 9.5.6.
Hi @lneumeier , cool that you will still fix some issues.
Our current license is valid until the end of November.
Could you consider keeping the license valid even if the new version was released after the license expires (as I assume no license update can be done now that the end of OEDT is near)?

**Comment by Ton-Pe** [comment]
> Will be fixed in OEDT 9.5.6.

Thank you so much!
